src/debug/README.md

```diff
@@ -1,3 +1,34 @@
 # Debugger
 
-Simplifies logging events to the console for debugging purposes.
\ No newline at end of file
+The Debugger module provides a centralized logging solution for SpringRoll games.
+
+```javascript
+Debugger.log('general', 'A general log');
+Debugger.log('log', 'Normal log');
+Debugger.log('warn', 'A warning');
+Debugger.log('debug', 'A debug message');
+Debugger.log('error', 'An error has occurred');
+```
+
+The debugger can be toggled off with the enable flag:
+
+```javascript
+Debugger.enable(false);
+```
+
+For more fine-grained control, you can set the minimum logging level:
+
+```javascript
+Debugger.minLevel('WARN');
+```
+
+Now, only `WARN` and `ERROR` level messages will be reported, and `GENERAL`, `DEBUG`, and `INFO` will not.
```

Comment:
Now, only `WARN` and `ERROR` level messages will report back, and `GENERAL`, `DEBUG`, and `INFO` will not.